orca-web/config/orca.yml

```diff
@@ -6,6 +6,7 @@ default:
     account: default
 
 front50:
+  enabled: true
```

Comment:
As far as I was able to verify, adding these flags here ensures default behavior is unaffected.